MEV_spec_RPC_v0_2.md

```diff
@@ -0,0 +1,141 @@
+        ],
+        "0x12ab34",
+        "0x0",
+        "0x0"
+    ]
+}' <url>
+
+# Response
+{
+    "id": 1337,
+    "jsonrpc": "2.0",
+    "result": "true"
+}
+```
+
+### Geth code changes
+
+[Add eth_sendBundle RPC · flashbots/mev-geth@8104d5d](https://github.com/flashbots/mev-geth/commit/8104d5d7b0a54bd98b3a08479a1fde685eb53c29)
+
+# eth_callBundle
+
+### Description
+
+Simulate a bundle of transactions at the top of a block.
+
+After retrieving the block specified in the `blockNrOrHash` it takes the same `blockhash`, `gasLimit`, `difficulty`, same `timestamp` unless the `blockTimestamp` property is specified, and increases the block number by `1`. `eth_callBundle` will timeout after `5` seconds.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| encodedTxs | `Array<Data>` | Array of signed transactions (`eth_sendRawTransaction` style, signed and RLP-encoded) |
+| blockNrOrHash	| `Quantity|string|Block Identifier` | Block number, or one of "latest", "earliest" or "pending", or a block identifier as described in {Block Identifier} |
+| blockTimestamp	|`Quantity`	|Block timestamp to be used in replacement of the timestamp taken from the parent block. |
+
+### Returns
+
+Map<`Data`, "error|value" : `Data`> - a mapping from transaction hashes to execution results with error or output (value) for each of the transactions
+
+### Example
+
+```bash
+# Request
+curl -X POST --data '{
+    "id": 1337,
+    "jsonrpc": "2.0",
+    "method": "eth_callBundle",
+    "params": [
+        [
+            "f9014946843b9aca00830493e094a011e5f4ea471ee4341a135bb1a4af368155d7a280b8e40d5f2659000000000000000000000000fdd45a22dd1d606b3782f2119621e928e32743000000000000000000000000000000000000000000000000000000000077359400000000000000000000000000000000000000000000000",
+            "f86e8204d085012a05f200830c350094daf24c20717f428f00d8448d74d67a77f67ceb8287354a6ba7a18000802ea00e411bcb660dd8d47717df89078d2e8160c08e7f11cb7ad0ee935e7436eceb32a013ee00a21b7fa0a9f9c1224d11261648191875d4633aed6003543ea319f12b62"
+        ],
+        "0x12ab34"
+    ]
+}' <url>
+
+# Response
+{
+    "id": 1337,
+    "jsonrpc": "2.0",
+    "result":
+        {
+            "0x22b3806fbef9532db4105475222983404783aacd4d865ea5dab76a84aa1a07eb" : {
+                "value" : "0x0012"
+            },
+            "0x489e3b5493af31d55059f8e296351b267720bc4ba7dc170871c1d789e5541027" : {
+                "value" : "0xabcd"
+            }
+        }
+}
+```
+
+## Geth code changes
+
+[Add eth_callBundle rpc method · flashbots/mev-geth@9199d2e](https://github.com/flashbots/mev-geth/commit/9199d2e13d484df7a634fad12343ed2b46d5d4c3)
+
+[Allow errors in the middle of eth_callBundle · flashbots/mev-geth@a99dfc1](https://github.com/flashbots/mev-geth/commit/a99dfc198817dd171128cc22439c81896e876619)
```

Comment:
Unclear what this is about, do we need to have this separate link? Shouldn't we have just one link with all changes?

Reply:
removed, thank you!